yorkie-intelligence/src/api/pr/views.py

```
@@ -0,0 +1,23 @@
+from fastapi.responses import StreamingResponse
+from langchain_core.output_parsers import StrOutputParser
+
+from src.common.llms import get_model
+from .config import issue_template_prompt
+
+
+router = APIRouter()
+
+
+@router.get("/streaming_pr")
+async def get_pr(query: str, llm=Depends(get_model)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)